public/calendar_reboot.js

```diff
@@ -35,9 +35,12 @@ function CheckIt() {
     
     var monthObjects;
     
+    this.backgroundActivityChange = new CustomEvent('backgroundActivityChange');
```

Comment:
The most recent and obvious example is `firebase.auth.onAuthStateChanged()`. You want to basically the same thing here. You have an object (`auth` or the `store`) that has some kind of hidden internal state that you don't really care about but you want to know when something important happens, such as logging in or out for `auth` or starting and finishing work for the `store`.